docs/software/spack.md

```diff
@@ -135,15 +157,19 @@ In addition to standard software packages, you can use Spack to install compiler
     ```
     spack compiler remove gcc -a
     ```
-    >Note that you can also remove intel compilers using `spack compiler remove intel -a`
+    ```{tip}
+    You can also remove intel compilers using `spack compiler remove intel -a`
+    ```
 
 5. Add the compiler you installed outside of the environment and install the compiler into the environment:
     ```
     spack compiler add $gcc_location
     spack install --add gcc@13.1.0
     ```
 
->Note that the install of the compiler within the environment is necessary as it will install all dependencies needed for the compiler. 
+```{important}
+The installation of the compiler within the environment is necessary as it will install all dependencies needed for the compiler. 
+```
 
 
 Once the compiler is added, you can install any subsequent packages using the compiler you've installed with the `%` operator. For example:
```

Comment:
I am seeing a a note-ish like statement beginning with: 
```
Installing compilers can take a long time, so make sure you request enough
```
Can you make this into an admonition? Maybe a tip?